src/core/serialize/src/main/scala/params/EstimatorArrayParam.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package org.apache.spark.ml.param
+
+import org.apache.spark.ml.Estimator
+import scala.collection.JavaConverters._
+
+/** Param for Array of Models. */
+class EstimatorArrayParam(parent: Params, name: String, doc: String, isValid: Array[Estimator[_]] => Boolean)
```

Comment:
Please make sure to add this to codegen so it can be correctly picked up

Reply:
done